src/mrgw.sh

```diff
@@ -11,7 +11,7 @@ port=$2
 
 shift 2
 
-run_root=$script_root/run/$name
+run_root=run/$name
```

Comment:
@cbodley I'm not sure that the way to go. We don't want to force users who use the different m* tools (mrun, mstart.sh, mrgw.sh) to run everything from the same directory. If that's a problem with cmake, we may need to add new configurables, either via a config file, or via yet another env variable.